subprocess/build.gradle

```diff
@@ -86,8 +90,8 @@ dependencies {
 //        runtimeOnly project(":debugger")
 //    }
 
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.11.4'
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

Comment:
See https://docs.gradle.org/current/userguide/upgrading_version_8.html#test_framework_implementation_dependencies

For some reason it didn't work with the previous junit version.